lib/esextractor.cpp

```diff
@@ -20,101 +20,105 @@
 #include "esenalstream.h"
 #include "eselogger.h"
 
-#include <assert.h>
+#include <memory>
+#include <cassert>
 
-ESExtractor::ESExtractor ():
-m_stream (nullptr)
+#if (__cplusplus < 201402L)
```

Comment:
This is a good catch, do we have testing for C++11 compilers? That's what the CTS is stuck on for a while yet.